src/figma/lightTheme.json

```diff
@@ -25,19 +25,19 @@
       "description": "For \"pressed\" states that use background-alternative."
     },
     "alternative-hover": {
-      "value": "#e7ebee",
+      "value": "#edeff1",
       "type": "color",
       "parent": "Theme Colors/Light mode",
       "description": "For \"hover\" states that use background-alternative."
     },
     "alternative-pressed": {
-      "value": "#dbe0e6",
+      "value": "#dfe0e2",
       "type": "color",
       "parent": "Theme Colors/Light mode",
       "description": "For \"pressed\" states that use background-alternative."
     },
     "hover": {
-      "value": "#0000000a",
+      "value": "#00000005",
```

Comment:
Updating to original colors in prod https://github.com/MetaMask/design-tokens/blob/07764c9f0098b39c3df09914164ee693c689593e/src/figma/tokens.json#L830-L861